application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java

```diff
@@ -35,67 +39,67 @@
 
     private boolean require_endpoint_verification;
 
-    public boolean isEnabled()
+    public final boolean isEnabled()
     {
         return enabled;
     }
 
-    public void setEnabled(boolean enabled)
+    public final void setEnabled(final boolean isEnabled)
     {
-        this.enabled = enabled;
+        this.enabled = isEnabled;
     }
 
-    public String getKeystore()
+    public final String getKeystore()
     {
         return keystore;
     }
 
-    public void setKeystore(String keystore)
+    public final void setKeystore(final String aKeystore)
     {
-        this.keystore = keystore;
+        this.keystore = aKeystore;
     }
 
-    public String getKeystorePassword()
+    public final String getKeystore_password()
```

Comment:
Why are you changing to underscore here? Can't we use the same as for everything else? like keyStorePassword? It should be possible to solve parsing from/to json/yaml with annotations.

Reply:
To keep to "standard" of other variables named this way in the class.

Reply:
It looks odd with an underscore, camel case is preferred. But if there is a good reason for an exception I would not go out of my way to change.

Reply:
Will need to be reworked if changing. Separate issue.